bpdv3/Assets/Scripts/UI/GameUI.cs

```diff
@@ -179,22 +173,64 @@ void Grading(float value)
             }
         }
     }
-    
+
+    private void ComboMetre()
+    {
+        if (playerMoveScript.isMoving)
+        {
+            if (!playerMoveScript.onBeat)
+            {
+                slider.value -= speed * Time.deltaTime;
+                if (slider.value <= sliderStartValue - 20)
+                {
+                    //playerScript.hitBeat = false;
```

Comment:
You have deleted lots of zombie code, this can be deleted too. https://zombiecodekill.com/2014/01/09/zombie-code/#:~:text=Zombie%20code%20is%20code%20that,code%20that%20is%20commented%20out.